src/components/common/container.module.scss

```diff
@@ -18,4 +18,28 @@
   & .container-narrow {
     max-width: 42rem;
   }
+  &.text-heavy {
+    margin: 5em auto;
+    @media (min-width: $viewport-lg) {
+      font-size: 22px;
+      line-height: 1.5;
+    }
+    h2 {
+      font-size: 30px;
```

Comment:
Same question [as above](https://github.com/COVID19Tracking/website/pull/535/files#diff-568a46f3cee58336c4574d08e1c143f5R24) regarding relative font-sizing!

Reply:
fixed in bdfd3485057bbe93c14f4b1ec4255617e5b139d1